opensuse_leap/15.0/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM opensuse/leap:15.0
+MAINTAINER Alexander V. Tikhonov <avtikhon@gmail.com>
+
+# Enable extra repositories
```

Comment:
This hunk previously installs necessary tools and then enable repositories. After some time extra repositories were removed, but the comment still states that we enable them here.

I would merge curl / wget installing into 'zypper install' below.

Reply:
Made as suggested.